node/v2/handler.js

```diff
@@ -143,7 +143,8 @@ TChannelV2Handler.prototype.handleFrame = function handleFrame(frame, callback)
 TChannelV2Handler.prototype.handleInitRequest = function handleInitRequest(reqFrame, callback) {
     var self = this;
     if (self.remoteHostPort !== null) {
-        return callback(new Error('duplicate init request')); // TODO typed error
+        return callback(errors.TChannelInitProtocolError(
```

Comment:
awesome, glad you implemented these TODOs in passing.  If you wanted to, this error conversion could separate out first since it won't break tests.